virt_who/testing.py

```diff
@@ -1124,11 +1124,18 @@ def vw_start_thread(
         return data, tty_output, rhsm_output
 
     def vw_start(self, cli=None, timeout=900, exp_send=1, exp_loopnum=0, oneshot=False, event=None, web_check=True, exp_error=False):
-        for i in range(3):
+        for i in range(4):
             data, tty_output, rhsm_output = self.vw_start_thread(cli, timeout, exp_send, exp_loopnum, oneshot, event, exp_error)
             if data['is_429'] == "yes":
                 wait_time = 60*(i+3)
-                logger.warning("429 code found, try again after %s seconds..." % wait_time)
+                logger.warning("429 code found, re-register virt-who host and try again after %s seconds..." % wait_time)
+                register_config = self.get_register_config()
+                register_type = register_config['type']
+                self.runcmd("\cp -f /etc/rhsm/rhsm.conf /root/rhsm429.conf", self.ssh_host())
```

Comment:
Is there something wrong when we run this as below? (if we remove the `\` in the start of the command) 

`self.runcmd("cp -f /etc/rhsm/rhsm.conf /root/rhsm429.conf", self.ssh_host())`

Reply:
If directly using "cp -f" when the target directory already contains the copy file, maybe it will still ask to confirm the action with "Y/N", because some systems set cp='cp -i' as default. "\cp -f" can fix the issue.
```
# alias
alias cp='cp -i'
```

Reply:
make sense :)